flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/Airbus.xml

```diff
@@ -444,129 +444,167 @@
             <SEQ1_POWERED>1</SEQ1_POWERED>
             <SEQ2_POWERED>1</SEQ2_POWERED>
         </DefaultTemplateParameters>
-
```

Comment:
**question:** Why remove the empty lines ?

Reply:
I think I reasoned that the `<!-- comment -->` would be enough spacing. I'm all for spacing and readability, but there is both patterns in the prioject.

Reply:
@Benjozork Added back spacing.